tests/test_entity_schema.py

```diff
@@ -0,0 +1,134 @@
+import os
+
+import pytest
+
+from whylabs_toolkit.helpers.schema import (
+    ColumnsClasses,
+    ColumnsDiscreteness,
+    UpdateColumnClasses,
+    UpdateColumnsDiscreteness,
+    UpdateEntityDataTypes
+)
+
+ORG_ID = os.environ["ORG_ID"]
+DATASET_ID = os.environ["DATASET_ID"]
+
+
+def test_change_columns_input_output() -> None:
+    classes = ColumnsClasses(
+        inputs=["temperature"],
+        outputs=["prediction_temperature"]
+    )
+
+    update_entity = UpdateColumnClasses(
+        classes=classes,
+        dataset_id=DATASET_ID,
+        org_id=ORG_ID
+    )
+
+    update_entity.update()
+
+    assert update_entity.current_entity_schema["columns"]["temperature"]["classifier"] == "input"
+    assert update_entity.current_entity_schema["columns"]["prediction_temperature"]["classifier"] == "output"
+
+    correct_classes = ColumnsClasses(outputs=["temperature"])
+
+    update_entity = UpdateColumnClasses(
+        classes=correct_classes,
+        dataset_id=DATASET_ID,
+        org_id=ORG_ID
+    )
+
+    update_entity.update()
+
+    assert update_entity.current_entity_schema["columns"]["temperature"]["classifier"] == "output"
+
+
+def test_change_columns_discreteness() -> None:
+    columns_discrete = ColumnsDiscreteness(
+        discrete=["prediction_temperature"],
+        continuous=["temperature"]
+    )
+
+    update_discreteness = UpdateColumnsDiscreteness(
+        dataset_id=DATASET_ID,
+        classes=columns_discrete,
+        org_id=ORG_ID
+    )
+
+    update_discreteness.update()
+
+    assert update_discreteness.current_entity_schema["columns"]["temperature"]["discreteness"] == "continuous"
+    assert update_discreteness.current_entity_schema["columns"]["prediction_temperature"][
+               "discreteness"] == "discrete"
+
+    columns_discrete = ColumnsDiscreteness(
+        discrete=["temperature"],
+        continuous=["prediction_temperature"]
```

Comment:
what happens if you set the same column as discrete and continuous?

Reply:
good point, will add this test